rock.osdeps

```diff
@@ -265,7 +265,9 @@ python-pexpect:
     opensuse: python-pexpect
 
 python-msgpack:
-    debian,ubuntu: python-msgpack
+    debian,ubuntu:
+        "18.04,18.10,19.04,19.10": python-msgpack
+        default: nonexistent
```

Comment:
Maybe install from pip instead?

Reply:
It's not that the package does not exist, it is that it's only available for Python 3.

I'd like to leave the Python 2 / Python 3 migration strategy to the people that actually are dealing with Python, @2maz ?

Reply:
> it is that it's only available for Python 3

That's my point... Can't you install the python2 version from pip?

Reply:
I don't want to build installing a python2 package from pip within the rock-core package set, unless there's something stepping in (i.e. not me) that vouches and maintains it.

I know what mixing versions of stuff does - in both compiled and dynamic languages - and I don't want to be responsible for it.

Reply:
> I don't want to be responsible for it

Fair enough